config.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type config struct {
+	DepWarnning  string
+	Script       []string `yaml:"script"`
+	Build        []string `yaml:"build"`
+	IgnoredItems []string `yaml:"ignore"`
+	Verbose      bool     `yaml:"verbose"`
+}
+
+func parseConfig() (config, error) {
```

Comment:
Could you also move the tests that go along with this function to a `config_test.go` file so that we can keep track of them?